index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>KIPCHOGE KEINO'S TRIBUTE</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body id = "main">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <h1 id = "title">ELIUD KIPCHOGE</h1>
+    <p>The man who did the impossible</p>
+    
+    <div id = "img-div">
+     <img src="https://media.cnn.com/api/v1/images/stellar/prod/210808193536-eliud-kipchoge-olympic-marathon.jpg?q=x_3,y_145,h_1423,w_2529,c_crop/h_720,w_1280" alt="Kipchoge picture" id = "image">
+     <p id = "img-caption"> Kipchoge celebrates his Olympic marathon victory in Sapporo.</p>
+    </div>
+    
+    
+    <section id = "tribute-info">
+
+      <h3>Here's a list of Eliud Kipchoge's achievements:</h3>
+      <ul id = "list-items">
+      <li><strong>2</strong> - World titles: Kipchoge won the 2003 junior world cross country title and 5000m gold at the 2003 world athletics championships.</li>
+      <li><strong>2</strong> - Marathon losses: The Kenyan finished second at the 2013 Berlin marathon and 8th at the 2020 London marathon</li>
+      <li><strong>4</strong> - Berlin Marathon titles: Kipchoge won the Berlin race in 2015, 2017, 2018 and 2022.</li>
+      <li><strong>4</strong> - Olympic Games appearances</li>
+      <li><strong>4</strong> - Olympic medals: Kipchoge clinched 5000m bronze at Athens 2014, 5000m silver at Beijing 2008 and claimed marathon gold at Rio 2016 and at Tokyo 2020.</li>
+      <li><strong>4</strong> - London marathon titles: He won the London race in 2015, 2016, 2018 and 2019.</li>
+      <li><strong>6</strong> - Medals won at world athletics championships, world cross country championships and world indoor championships.</li>
```

Comment:
- [ ] Please your comment is not clear enough. What tag should be avoided? Kindly ensure that your review comments are accurate, clear and easy to understand. :wink: This will help the student grasp the concepts you are trying to put across for easy implementation. :wink: 
- [ ] Kindly include an explanation, a 'why', for each requested change. This will help the student know exactly what is requested and why it should be done in the way you are requesting it. :pray: